ecosystem/sep-0006.md

```diff
@@ -1044,13 +1078,25 @@ Example response:
       "id": "82fhs729f63dh0v4",
       "kind": "withdrawal",
       "status": "completed",
-      "amount_in": "500",
+      "amount_in": "510",
       "amount_out": "495",
```

Comment:
This should be 485.
amount_out = amount_in - amount_fee - refunds.amount_refunded - refunds.amount_fee
485 = 500 - 5 - 10 - 0